guides/release/testing/index.md

```diff
@@ -18,32 +18,45 @@ Writing tests is also a fun activity, a nice change of pace from delivering feat
 
 Every Ember app comes with [QUnit](http://qunitjs.com/) and [QUnit DOM](https://github.com/simplabs/qunit-dom). QUnit is a testing framework, and QUnit DOM is a library that helps you **write tests that are concise and readable**.
 
-To see the power of QUnit DOM, consider this code snippet. It checks whether our button component shows the right text and the right CSS attribute.
+To see the power of QUnit DOM, consider this code snippet. It checks whether our button component shows the right label and the right attributes.
 
-```javascript {data-filename=tests/integration/components/simple-button/component-test.js}
+```javascript {data-filename=tests/integration/components/simple-button-test.js}
```

Comment:
File paths now assume Glimmer components.

Reply:
The comment confuses me a little bit 😅 How does Glimmer affect the file path?

Reply:
My bad. I could have explained it better.  It has been a while since I last worked on this, so I might be remembering it incorrectly.

When I ran the CLI command with the `-gc` option, the rendering test was created with the component's name preprended to `-test.js`, rather than with the filename `component-test.js` that I had been familiar with in classic Ember.

Does that address your question?